sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -228,7 +228,7 @@ inline __SYCL_ALWAYS_INLINE void joint_matrix_load(
         Ptr, stride, __spv::MatrixLayout::ColumnMajor,
         spv_scope_traits<Group>::value);
     break;
-  case sycl::ext::intel::experimental::matrix::layout::packed:
+  case sycl::ext::intel::experimental::matrix::layout::ext_intel_packed:
```

Comment:
we should not need to specify namespace anymore as this is now in the same namespace as joint_matrix